ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -0,0 +1,136 @@
+chain:
+  as: hypershift-kubevirt-create
+  steps:
+  - as: create-hostedcluster
+    cli: latest
+    env:
+    - name: HYPERSHIFT_NODE_COUNT
+      default: "3"
+      documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_BASE_DOMAIN
+      default: "ci.azure.devcluster.openshift.com"
+      documentation: "The cluster's FQDN will be a subdomain of the base domain."
+    commands: |-
+      set -exuo pipefail
+
+      # Deploy HyperConverged custom resource to complete kubevirt's installation
+      oc create -f - <<EOF
+      apiVersion: hco.kubevirt.io/v1beta1
+      kind: HyperConverged
+      metadata:
+        name: kubevirt-hyperconverged
+        namespace: openshift-cnv
+      EOF
+      
+      oc wait HyperConverged -n openshift-cnv kubevirt-hyperconverged --for=condition=Available --timeout=15m           
+
+      KUBEVIRT_CLUSTER_NAME="kv-nested"
+      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|cut -d- -f-2)"
+
+      echo "$(date) Creating HyperShift cluster ${KUBEVIRT_CLUSTER_NAME}"
+      /usr/bin/hypershift create cluster kubevirt \
+        --name ${KUBEVIRT_CLUSTER_NAME} \
+        --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
```

Comment:
It took me awhile to get the kubevirt/hypershift presubmits working because some operators didn't come online due to lack of resources.

I ended up with a configuration that required 2 replicas with 6Gi of memory [1]. For the conformance tests, I'd start with that setup, and expand to replica `3` if you find you're running out of capacity. If you don't explicilty ask for 6Gi of memory, the default is 4Gi. The monitoring stack will fail to come online with 4Gi.

1. https://github.com/openshift/release/blob/1bf15351105cb137179a33b9282351829b4455da/ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml#L158

Reply:
Updated the node count to 2, each with memory of 6Gi.